presto-main/src/main/java/io/prestosql/execution/buffer/BroadcastOutputBuffer.java

```diff
@@ -104,7 +104,7 @@ public double getUtilization()
     @Override
     public boolean isOverutilized()
     {
-        return memoryManager.isOverutilized();
```

Comment:
add a comment with rationale.

Why check for `canAddPages`. If we can't add pages, it seems even more overutilzed

Reply:
I think that `canAddPages` is different from `BUFFER_FULL`. Since the `setNoMorePages` is done only when all the drivers are completed in a task, indicating that the buffer will not get any new pages from now.